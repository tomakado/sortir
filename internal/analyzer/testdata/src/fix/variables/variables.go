package variables

var (
	zebra = "z"
	apple = "a" // want "variable/constant declarations are not sorted"
	banana = "b"
)

var (
	z int
	x int // want "variable/constant declarations are not sorted"
	y int
)