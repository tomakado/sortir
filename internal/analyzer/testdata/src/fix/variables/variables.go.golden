package variables

var (
	apple  = "a" // want "variable/constant declarations are not sorted"
	banana = "b"
	zebra  = "z"
)

var (
	x int // want "variable/constant declarations are not sorted"
	y int
	z int
)